functions/zoom-uploader.py

```diff
@@ -228,8 +230,21 @@ def workflow_definition_id(self):
     def s3_filenames(self):
         if not hasattr(self, "_s3_filenames"):
             self._s3_filenames = {}
-            for view, file in self.data["s3_files"].items():
-                self._s3_filenames[view] = [data["filename"] for data in file["segments"]]
+            for view, file_info in self.data["s3_files"].items():
+                segments = file_info["segments"]
+
+                # skip false starts
+                if segments[0]["ffprobe_seconds"] < MINIMUM_DURATION * 60:
+                    # first segment for each view might not be the overall
+                    # first segment, need to check
```

Comment:
Can you elaborate on this some more?

Reply:
You might have.
2pm - 15s
- speaker

2:01pm - 1m
- speaker
- shared screen

2:15pm - 1 hour
- speaker
- shared screen

If you remove the first segment in each view that is <2 minutes then your segments will not line up.